spec.html

```diff
@@ -27753,6 +27762,10 @@ <h1>Extended Years</h1>
                     <td>+002009-12-15T00:00:00Z</td>
                     <td>2009 A.D.</td>
                   </tr>
+                  <tr>
+                    <td>+275760-09-13T00:00:00Z</td>
+                    <td>275760 A.D.</td>
+                  </tr>
```

Comment:
What is the purpose of the addition of these extended years? It doesn't look like they are boundary cases, due to the more extremely values preceding and following them.

Reply:
Oh I see, these are what you are separately proposing to be the new extreme permitted values, right?